tools/segemehl/segemehl.xml

```diff
@@ -14,7 +14,7 @@
         ## prepare segemehl index if no reference genome is supplied
         #if $refGenomeSource.genomeSource == "history":
             mkdir ./temp_index/ &&
-	    #set $temp_index = './temp_index/'
+	    #set $temp_index = './temp_index/$refGenomeSource.own_reference_genome'+'.idx'
```

Comment:
I think we can just use some temp name like `./temp_index/tmp.idx` or is the name really interesting. `$refGenomeSource.own_reference_genome` does probably not work, as this is the entire path to a file.